code/modules/mob/living/simple_animal/abnormality/zayin/bottle.dm

```diff
@@ -61,19 +61,56 @@
 			//Drowns you like Wellcheers does, so I mean the code checks out
 			for(var/turf/open/T in view(7, src))
 				new /obj/effect/temp_visual/water_waves(T)
-			to_chat(user, "<span class='userdanger'>The room is filling with water! You're going to drown!</span>")
 			playsound(get_turf(user), 'sound/abnormalities/bottle/bottledrown.ogg', 80, 0)
 			icon_state = "bottle3" //cake all gone
 
+			/* "I get it now. There's no reason to have any emotions or a heart."
+			"Abandon reason and protect Catt! That's how you survive in this wonderland!"
+			Catt was a low level agent with temperance and fortitude. She sought to abandon that temperance in exchange for justice.
+			Thus, bottle will check for those. Overall low level, high temperance, high fortitude.
+			Keep in mind that this event can happen ONCE PER ROUND. The spoon is already jokingly called the protagonist weapon, so it can bend the rules a bit, right?*/
+
 			new /obj/item/ego_weapon/eyeball(get_turf(user))
 
+			var/fortitude = get_attribute_level(user, FORTITUDE_ATTRIBUTE)
+			var/prudence = get_attribute_level(user, PRUDENCE_ATTRIBUTE)
+			var/temperance = get_attribute_level(user, TEMPERANCE_ATTRIBUTE)
+			var/justice = get_attribute_level(user, JUSTICE_ATTRIBUTE)
+			var/goal_damage = 0
+			if(temperance >= (fortitude + prudence + justice) / 1.5) // If your temperance is at least twice your average stat, you aren't hurt, but lose temperance.
+				to_chat(user, "<span class='userdanger'>The room is filling with water... but you feel oddly unconcerned.</span>")
+				user.adjust_attribute_level(TEMPERANCE_ATTRIBUTE, 20 - temperance)
+				// This is a PERMANENT stat change, VERY significant. But it can happen only once per round. You're The Protagonist, after all.
```

Comment:
Permanent stat reduction that happens EARLY and it's the stat that can be regained. Not as bad as you're making it sound but also not an issue I think.

Reply:
I know, but I wanted to make clear that I know it's far, far beyond the norm for an abnormality to affect/reduce someone's base stats rather than applying a bonus, and that I feel the degree of which is reasonable.